tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/tasks/GenerateAndBuildTask.ts

```diff
@@ -0,0 +1,70 @@
+import {
+    addFileLog,
+    GenerateAndBuildInput,
+    GenerateAndBuildOptions, getGenerateAndBuildOutput,
+    getTask, removeFileLog, requireJsonc,
+    runScript
+} from '@azure-tools/sdk-generation-lib';
+import fs from 'fs';
+import path from 'path';
+import { DockerTaskEngineContext } from '../DockerTaskEngineContext';
+import { SDKGenerationTaskBase, TaskType } from './SDKGenerationTaskBase';
+
+export class GenerateAndBuildTask implements SDKGenerationTaskBase {
+    taskType: TaskType;
+    order: number;
```

Comment:
Not sure why we need it.  Is it the information defined in external config?

Reply:
It's the order for pre-defined task. If this information doesn't exist, we need to add some `if/else` in executing the tasks, so I think the order is useful. I don't want to defined it in external config because they are predefined task, and I don't want to break the original configuration of sdk automation too much.